plugins/available/wttr.plugin.bash

```diff
@@ -0,0 +1,39 @@
+function wttr() {
+if [ -n "$1" ]; then
+    if [ "$(ping -c 1 wttr.in)" ]; then
+        if [ "$(which curl 2>/dev/null)" ]; then
+            CITY="$(curl -s ipinfo.io/city)"
+            echo "Weather of $1"
+			      echo " "
+            curl -s "wttr.in/$1?Q0"
+        elif [ "$(which wget 2>/dev/null)" ]; then
+            CITY="$(wget -qO- ipinfo.io/city)"
+            echo "Weather of $1"
+			      echo " "
+            wget -qO- "wttr.in/$1?Q0"
+        fi
+        echo " "
+    else
+        echo "Can't connect to wttr.in"
+        exit 1
+    fi
+else
+    if [ "$(ping -c 1 wttr.in)" ]; then
```

Comment:
A lot of this code seems to be redundant. The code is covering 4 different cases:
* curl or wget?
* Parameter provided (`$1`) or auto-detect the city

I'm pretty sure that the same can be achieved without duplicating the code several times. Can you please try to clean this up and simplify the code so that the code is easier to read and to maintain?